app/src/androidTest/java/com/rootstrap/android/tests/SignInActivityTest.kt

```diff
@@ -42,7 +44,7 @@ class SignInActivityTest : BaseTests() {
 
         activity.runOnUiThread {
             val current = currentActivity()
-            assertEquals(ProfileActivity::class.java.name, current::class.java.name)
+            assertEquals(MainActivity::class.java.name, current::class.java.name)
```

Comment:
Does this test work? Because the test scenario is the OnBoardingActivity and here we are checking for MainActivity

Reply:
yes that was changed too.

Reply:
yes, this is only to check if the app navigate to the MainActivity

Reply:
Good! Thanks for the explanation